boundary_layer/schemas/dag.py

```diff
@@ -35,9 +35,15 @@ class ReferenceSchema(OperatorSchema):
     target = fields.String(required=True)
 
 
+class BatchingSchema(StrictSchema):
```

Comment:
minor maybe, but you could do this with just a single optional `batch_size` argument in the generator schema, i think?  and if that argument is missing then you implicitly assume that `enabled == False`.  This would also simplify the part above where you say you think there would be a better way...

Reply:
See my comment in `generator_operator.j2` for my logic here.